Resources/Locale/en-US/accent/french.ftl

```diff
@@ -0,0 +1,131 @@
+accent-french-replaced-1 = yes
+accent-french-replacement-1 = oui
+
+accent-french-replaced-2 = hi
+accent-french-replacement-2 = salut
+
+accent-french-replaced-3 = hello
+accent-french-replacement-3 = allo
+
+accent-french-replaced-4 = bye
+accent-french-replacement-4 = au revoir
+
+accent-french-replaced-5 = goodbye
+accent-french-replacement-5 = au revoir
+
+accent-french-replaced-6 = the
+accent-french-replacement-6 = la
+
+accent-french-replaced-7 = is
+accent-french-replacement-7 = est
+
+accent-french-replaced-8 = bread
+accent-french-replacement-8 = baguette
+
+accent-french-replaced-9 = very
+accent-french-replacement-9 = très
+
+accent-french-replaced-10 = good
+accent-french-replacement-10 = bien
+
+accent-french-replaced-11 = great
+accent-french-replacement-11 = super
+
+accent-french-replaced-12 = beautiful
+accent-french-replacement-12 = beau
+
+accent-french-replaced-13 = music
+accent-french-replacement-13 = musique
+
+accent-french-replaced-14 = greetings
+accent-french-replacement-14 = bonjour
+
+accent-french-replaced-15 = france
```

Comment:
This is overdoing it. This would change "I'm from France" to "I'm from la france" which a speaker would never do since it's "de France"